src/web-ui/src/main.js

```diff
@@ -72,7 +72,7 @@ require('dotenv').config()
 
 Vue.config.productionTip = false
 
-Logger.LOG_LEVEL = 'DEBUG'
+//Logger.LOG_LEVEL = 'DEBUG'
```

Comment:
Why this change?

Reply:
We were just trying something for the image preview to show up. But it did not work and I forgot to uncomment this piece. This needs to be uncommented.